web/src/main/ui/app/components/flows-new/manage-flows/manage-flows.component.ts

```diff
@@ -78,15 +78,15 @@ export class ManageFlowsComponent implements OnInit {
         .subscribe(() =>  this.manageFlowsService.getFlowById(runObject.id).subscribe( poll => {
           const flowIndex = this.flows.findIndex(flow => flow.id === runObject.id);
           this.flows[flowIndex] = Flow.fromJSON(poll);
-          this.flowsPageUi.renderRows();
-          if (this.flows[flowIndex].flow.latestJob && this.flows[flowIndex].flow.latestJob.status) {
-            let runStatus = this.flows[flowIndex].flow.latestJob.status.replace('_', ' ');
-            runStatus = this.flows[flowIndex].flow.latestJob.status.replace('-', ' ');
-            runStatus = this.flows[flowIndex].flow.latestJob.status.split(' ');
+          if (this.flows[flowIndex].latestJob && this.flows[flowIndex].latestJob.status) {
+            let runStatus = this.flows[flowIndex].latestJob.status.replace('_', ' ');
+            runStatus = this.flows[flowIndex].latestJob.status.replace('-', ' ');
```

Comment:
What is the point to override this again?

Reply:
Job status messages from the backend are currently not standardized. Please see DHFPROD-2283
https://project.marklogic.com/jira/browse/DHFPROD-2283